### PR TITLE
samples: bluetooth: nrf_dm: update figure in the documentation.

### PR DESCRIPTION
**Pull request by maje-emb**
Update the Synchronization process figure to match the guidelines.

Ref: NCSDK-15038

Signed-off-by: Marcin Jeliński <marcin.jelinski@nordicsemi.no>

### CODE REVIEW

**Review by peknis — APPROVED** [review]

**Comment by peknis** [comment]
The compliance failure is a mystery to me. Maybe it has something to do with this:

Doxygen is getting updated to 1.9.4 because of issues in doc build in main. (
[Kuroś, Andrzej: Cannot build documentation https://github.com/nrfconnect/sdk-nrf/pul…](https://teams.microsoft.com/l/message/19:a44c20c9d6f349fd9adf98a517fe32cb@thread.skype/1651821807766?tenantId=28e5afa2-bf6f-419a-8cf6-b31c6e9e5e8d&groupId=bbf3a6ae-b458-4b7b-9c01-2d2881d8b2fb&parentMessageId=1651821807766&teamName=NCS&channelName=NCS%20CI&createdTime=1651821807766)

posted in NCS / NCS CI at Friday, May 6, 2022 9:23:27 AM ). PR open here: https://github.com/nrfconnect/sdk-nrf/pull/7515.

**Review by greg-fer — COMMENTED** [review]
Can be also rendered with the following formatting:

```suggestion
.. msc::
   hscale = "1.3";
   DevA [label="Device A"],DevB [label="Device B"];
   |||;
   DevA rbox DevB [label="Synchronization starts"];
   |||;
   DevA=>DevB [label="ADV_IND"];
   DevA<=DevB [label="SCAN_REQ"];
   DevA=>DevB [label="SCAN_RSP, Manufacturer-specific data"];
   |||;
   DevA rbox DevB [label="Synchronization ends"];
   |||;
   DevA box DevA [label="Reflector"],DevB box DevB [label="Initiator"];
   |||;
   DevA<=>DevB [label="Ranging"];
```

We save having to upload two additional files this way.

**Review by greg-fer — CHANGES_REQUESTED** [review]
The images need to be moved to ncs/nrf/doc/nrf/images directory. This is because we do not add images for samples in places other than this directory. Linking from the tag does not require providing the path, only file name should work.

**Comment by maje-emb** [comment]
I have used the rendered diagram.

File: samples/bluetooth/nrf_dm/README.rst [hunk 1]
```diff
@@ -34,10 +34,23 @@ The synchronisation step has two goals:
...
+   DevA<=DevB [label="SCAN_REQ"];
+   DevA=>DevB [label="SCAN_RSP, Manufacturer-specific data"];
+   |||;
+   DevA rbox DevB [label="Synchronization ends"];
+   |||;
+   DevA box DevA [label="Reflector"],DevB box DevB [label="Initiator"];
+   |||;
+   DevA<=>DevB [label="Ranging"];
+
+The chart shows the sequence of synchronisation process.
```
Comment (peknis) [inline samples/bluetooth/nrf_dm/README.rst:53]:
```suggestion
The chart shows the sequence of the synchronization process.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by greg-fer — APPROVED** [review]

**Review by peknis — APPROVED** [review]

**Pull request merged**